packages/support/resources/js/sortable.js

```diff
@@ -8,6 +8,10 @@ export default (Alpine) => {
             draggable: '[x-sortable-item]',
             handle: '[x-sortable-handle]',
             dataIdAttr: 'x-sortable-item',
+            animation: 300,
```

Comment:
Have you tried different values, like 200, to make it feel a little snappier?

Reply:
Yeah, I played a little.
I enjoy where it's at, 200.

What do you think of keeping a default, but also customizable?

Reply:
It's currently at 300, not 200. 😄
Since this will also be used for ordering table rows (right?), I figured 300 might be a little too long.

Hmm, just having a sensible default should work I think.
How would you allow users to customize this?

Reply:
eheheh my typo!

So, I went ahead and implemented the custom animation time. I followed the naming sortable has - as in "duration" and not "durationTime".

Let me know what you think. 😄 

![image](https://github.com/filamentphp/filament/assets/26031459/580afd04-6aa4-403d-ba88-be009e8b78a0)

Reply:
Could we have a GitPod setup?

It would come in handy for trying PR's in 1 click. Similarly to the demo https://www.youtube.com/watch?v=IZLcKZzoqW8

Reply:
I tried it, but it's not as flexible. I don't need one, I thought you (or the team) could benefit from tinkering with the UI, with a single click to visit the PR.

I could help by bringing that up.

Does it make sense for you? (whether one or the other)